pkgs/by-name/cd/cdecl/package.nix

```
@@ -0,0 +1,68 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bison,
+  flex,
+  readline,
+  ncurses,
+  gcc
+}:
+
+stdenv.mkDerivation {
+  pname = "cdecl";
+  version = "2.5-unstable-2024-05-07";
+
+  src = fetchFromGitHub {
+    owner = "ridiculousfish";
+    repo = "cdecl-blocks";
+    rev = "1e6e1596771183d9bb90bcf152d6bc2055219a7e";
+    hash = "sha256-5XuiYkFe+QvVBRIXRieKoE0zbISMvU1iLgEfkw6GnlE=";
+  };
+
+  buildInputs = [
+    gcc
+    bison
+    flex
+    readline
+    ncurses
+  ];
+
+  NIX_CFLAGS_COMPILE = "-DBSD -DUSE_READLINE";
+  NIX_LDFLAGS = "-lreadline";
+
+  makeFlags = [
+    "PREFIX=${placeholder "out"}"
+    "BINDIR=${placeholder "out"}/bin"
+    "MANDIR=${placeholder "out"}/man1"
```

```suggestion
    "MANDIR=${placeholder "out"}/share/man/man1"
```

What is CATDIR, by the way?

---

It's a make variable that is defaulted to `/usr/man/cat1`

From [Exploring the File System](https://www.krsaborio.net/linux/research/1995/install-guide-2.2.2/node116.html):

> `/usr/man/cat1` contains the formatted man pages for section 1